doc/source/user/tutorials_index.rst

```diff
@@ -5,12 +5,8 @@ NumPy Tutorials
 ################
 
 These documents are intended as an introductory overview of NumPy and its 
-features. For detailed reference documentation of the functions and
-classes contained in the package, see the :ref:`API reference <reference>`.
-
-.. toctree::
-   :maxdepth: 1
-
-   tutorial-svd
-   tutorial-ma
+features, as well as a few applications. You can find them 
+`in the NumPy Tutorials page <https://numpy.org/numpy-tutorials>`_.
 
+For detailed reference documentation of the functions and
+classes contained in the package, see the :ref:`API reference <reference>`.
```

Comment:
Hmm. This doesn't really flow well. Maybe something like

Tutorials about NumPy are hosted at https://numpy.org/numpy-tutorials. For how-tos, reference documents, and explanations see the ref:`API reference <reference>` documentation.

Reply:
Fixed, thanks!